simpleclient_spring_boot/src/main/java/io/prometheus/client/spring/boot/PrometheusMVCEndpoint.java

```diff
@@ -0,0 +1,36 @@
+package io.prometheus.client.spring.boot;
+
+import org.springframework.boot.actuate.endpoint.mvc.EndpointMvcAdapter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import io.prometheus.client.exporter.common.TextFormat;
+
+@ConfigurationProperties("endpoints.prometheus")
+public class PrometheusMVCEndpoint extends EndpointMvcAdapter {
```

Comment:
@nicolinux72 You may have noticed that this PR is addressing the same thing as #225 

I like how #225  implemented the `PrometheusMvcEndpoint` better by `extends AbstractEndpointMvcAdapter<PrometheusEndpoint> ` and the `invoke` implmentation (since it mirrors the HealthMvcEndpoint class in Spring).